ivy/functional/ivy/experimental/elementwise.py

```diff
@@ -1335,3 +1335,54 @@ def digamma(
     ivy.array([-0.7549271   0.92278427  0.9988394])
     """
     return ivy.current_backend(x).digamma(x, out=out)
+
+
+@handle_exceptions
+@handle_nestable
+@handle_out_argument
+@handle_array_function
+def sparsify_tensor(
+    x: Union[ivy.Array, ivy.NativeArray],
+    card: int,
+    /,
+    out: Optional[ivy.Array] = None,
+) -> ivy.Array:
+    """
+    Zeros out all elements in the `tensor` except `card` elements with maximum absolute
+    values.
+
+    Parameters
+    ----------
+    x : Union[ivy.Array, ivy.NativeArray]
+        Tensor to be sparsified
+    card : int
```

Comment:
```suggestion
    card
```
Already there in the typehints.